src/Group.php

```diff
@@ -123,15 +123,6 @@ public function getMembers(): array
         return $this->members;
     }
 
-    /**
-     * Set the value of members
-     * @param User $member
-     */
-    public function setMembers(User $member): void
```

Comment:
This was a public function, so it removes `setMembers` from the available external interface of this class.
Is anything using `setMembers` from the Moodle integration?
Is the functionality of `setMembers` a requirement for this SDK?